app/src/store/ThunkAlerts.re

```diff
@@ -0,0 +1,19 @@
+open AppStore;
+open StoreAlerts;
+open Source;
+
+let showAlert = (severity, message, dispatch, state, _source: source) => {
+  let alert = {severity, message, id: state.alerts.idCounter};
```

Comment:
Instead of depending on the store keep the id handling, we can use the Random module from Reason and create it locally. This would simplify things.

Reply:
Ha, was thinking of this before, but wanted the alert system to be 1000% robust, you know, in case the random ids clash ; P

Will try it out!